scripts/build_tag.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
```

Comment:
Please add [our standard copyright header](https://github.com/apple/coremltools/blob/8b7ebd851a93917bd2985982122d5154601e9690/setup.py#L3-L6) to this file. Thanks.

Reply:
fixed